### PR TITLE
PIA-2257 xcframeworks script

### PR DESCRIPTION
**Pull request by chriskryn**
xcframeworks build automation script

### CODE REVIEW

File: .github/workflows/bank-sdk.build.xcframeworks.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+name: Build BankSDK XCFrameworks
+on:
+  push:
```
Comment (a-szotyori) [inline .github/workflows/bank-sdk.build.xcframeworks.yml:3]: We need also `workflow_call` as a trigger in `on` to make this callable from another workflow. https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Wow, Awesome! There is only a small thing related to making the workflow callable from another workflow.

File: .github/workflows/bank-sdk.build.xcframeworks.yml (hunk 1 above)
Comment (zladzeyka) [reply .github/workflows/bank-sdk.build.xcframeworks.yml:3]: like we have https://github.com/gini/gini-mobile-ios/blob/main/.github/workflows/bank-api-library.publish.docs.yml#L6

**Review by a-szotyori — APPROVED** [review]

**Review by zladzeyka — APPROVED** [review]
🎉👏🏻😊 Great job @chriskryn

**Pull request merged**